src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/GlobalEventRoutes.scala

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.bluebrain.nexus.kg.routes
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.marshalling.sse.EventStreamMarshalling._
+import akka.http.scaladsl.server.Directives.complete
+import akka.http.scaladsl.server.Route
+import akka.persistence.query._
+import akka.persistence.query.scaladsl.EventsByTagQuery
+import ch.epfl.bluebrain.nexus.iam.client.config.IamClientConfig
+import ch.epfl.bluebrain.nexus.kg.resources.Event.JsonLd._
+import ch.epfl.bluebrain.nexus.iam.client.types.{AccessControlLists, Caller, Permission}
+import ch.epfl.bluebrain.nexus.kg.config.AppConfig
+import ch.epfl.bluebrain.nexus.kg.directives.AuthDirectives._
+import ch.epfl.bluebrain.nexus.kg.persistence.TaggingAdapter
+
+class GlobalEventRoutes(acls: AccessControlLists, caller: Caller)(implicit as: ActorSystem, config: AppConfig)
+    extends EventCommonRoutes {
+
+  override val pq: EventsByTagQuery =
+    PersistenceQuery(as).readJournalFor[EventsByTagQuery](config.persistence.queryJournalPlugin)
+
+  private val read: Set[Permission]             = Set(Permission.unsafe("resources/read"), Permission.unsafe("events/read"))
```

Comment:
the permission needs to be `events/read` as with the other services.

Reply:
but wouldn't it make sense to allow both? if `resource/read` on `/org/project` gives you access to event stream for a project, shouldn't `resource/read` also give you access to full event stream?

Reply:
I think we need to guard event streaming with a separate permission (which is why in iam and admin to get the entire event log you need `events/read`).

For v1.1 we can consider applying different permissions and change the routes to use different paths... but for now, even if it's just for consistency, we should use `events/read` for the global event log.